zzre/game/systems/ui/ScrDeck.cs

```diff
@@ -443,10 +443,10 @@ private void CreateGridList(DefaultEcs.Entity entity, ref components.ui.ScrDeck
 
     private IEnumerable<InventoryCard> AllCardsOfType(in components.ui.ScrDeck deck) => deck.ActiveTab switch
     {
-        Tab.Items => deck.Inventory.Items,
-        Tab.Fairies => deck.Inventory.Fairies,
-        Tab.AttackSpells => deck.Inventory.AttackSpells,
-        Tab.SupportSpells => deck.Inventory.SupportSpells,
+        Tab.Items => deck.Inventory.Items.OrderBy(c => c.cardId.EntityId),
+        Tab.Fairies => deck.Inventory.Fairies.OrderBy(c => -c.level),
```

Comment:
nitpick: [`OrderByDescending`](https://learn.microsoft.com/de-de/dotnet/api/system.linq.enumerable.orderbydescending?view=net-8.0) would be slightly more clear

Reply:
Glad to know more api, fixed